src/serialized-utilities.h

```diff
@@ -0,0 +1,89 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2021 Intel Corporation. All Rights Reserved.
+
+#pragma once
+#include <string>
+#include <map>
+#include <types.h>
+#include "../third-party/json.hpp"
+
+using json = nlohmann::json;
```

Comment:
Not in an include... at least, not at global scope...

Reply:
👍